l10n_ch_payment_slip/views/account_invoice.xml

```diff
@@ -1,71 +1,74 @@
 <?xml version="1.0"?>
 <openerp>
-  <data>
+    <data>
 
-    <record id="action_slip_all" model="ir.actions.act_window">
-      <field name="name">Payment Slip</field>
-      <field name="res_model">l10n_ch.payment_slip</field>
-      <field name="domain">[('invoice_id', '=', active_id)]</field>
-    </record>
+        <record id="action_slip_all" model="ir.actions.act_window">
```

Comment:
Please don't change the indentation from 2 spaces to 4 spaces. That makes xml big for nothing and history is overrided.

Reply:
The indentation in this file was an absolute mess. I can change to have 2 spaces to avoid big files but there will still be indentation changes.